### ISSUE TITLE
Update the recurring contributions experience

### ISSUE THREAD

**Issue by Memo-Es** [opencollective/opencollective#6217]
The current experience with editing contributions feels broken, why?

**Currently, there are 2 ways to go to the page to manage contributions for individual users:**

Link in the user menu | Button to "Edit contribution" in the payment methods section inside the user's settings
-------------|--------------
<img width="1340" alt="Screen Shot on 2022-12-02 at 12-48-27" src="https://user-images.githubusercontent.com/49539954/206094585-42b613c8-f6b3-4115-89aa-1e82de5e0678.png"> | <img width="2560" alt="Screenshot 2022-12-02 at 12 29 17 (2)" src="https://user-images.githubusercontent.com/49539954/206095474-8df0a347-886d-499f-98b7-8659d4771387.png">

**Then, you land on a page that feels inside your public profile**

<img width="1340" alt="Screen Shot on 2022-12-02 at 12-48-27" src="https://user-images.githubusercontent.com/49539954/206096137-6cfe4778-7fd2-4a7e-8d96-c932c4411f89.png">

This needs to be revised and framed as a project, so I include  my thoughts here as a starting point:

1. This page presents admin rights to manage the users' contributions; the very word manage tells me this should be somewhere in the settings.
2. Each element to edit is displayed as a Collective/Profile in the platform. It should be rendered with a different interface to prevent confusion with it being a profile (as per #5680). It feels closer to a transaction than to a profile as a whole.

I'm not sure about the final solution, so I'm just leaving it here documented because it is also related to the transactions UX issue. (Issue pending)

**Comment by Betree** [comment]
I agree that the user's admin panel seems like the right place for such a feature to live.

Something we should be careful about, though: the interface when you're an admin of multiple profiles actually looks like that:

![image](https://user-images.githubusercontent.com/1556356/206185400-c3640c86-e58f-4479-bdd7-d8d3c37b516d.png)

You get all the profiles you're administrating on the left and can easily switch between one another. This has proved to solve a very common issue of users not being able to find the page to manage contributions for the organizations they're administrating. It's also very practical for us to have a single URL (https://opencollective.com/recurring-contributions) without any dynamic parameter (like the username) that we can use to link people to their recurring contributions.
